expo/expo/urls.py

```diff
@@ -3,21 +3,36 @@
 from django.conf.urls.static import static
 from cms.sitemaps import CMSSitemap
 
+from expo.local_settings import APP_DJANGOBB
+
 from django.contrib import admin
 admin.autodiscover()
 
-urlpatterns = patterns('',
-                       url(r'^', include('gbe.urls', namespace = 'gbe')), 
-                       url(r'^admin/', include(admin.site.urls)),
-                       url(r'^', include('ticketing.urls', namespace = 'ticketing')),
-                       url(r'^', include('scheduler.urls', namespace = 'scheduler')),
-                       url(r'^', include('gbe.report_urls', namespace = 'reporting')),
-                       url(r'^tinymce/', include('tinymce.urls')),
-                       url(r'^sitemap\.xml$', 'django.contrib.sitemaps.views.sitemap',
-                           {'sitemaps': {'cmspages': CMSSitemap}}),
-                       url(r'^hijack/', include('hijack.urls')),
-                       url(r'^', include('cms.urls'))
-) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+LOCAL_APPS_URLS = ()
+
+# The indentation on this is FUGLY and BAD, but PEP8
+if APP_DJANGOBB is True:
+    LOCAL_APPS_URLS = LOCAL_APPS_URLS+(url(r'^forum/',
```

Comment:
missing space around + operator. break line after first open paren if needed to respect 80 column limit